test/ex_machina/ecto_has_many_test.exs

```diff
@@ -7,6 +7,7 @@ defmodule ExMachina.EctoHasManyTest do
     schema "packages" do
       field :description, :string
       has_many :statuses, ExMachina.EctoHasManyTest.PackageStatus
+      has_many :invoices, ExMachina.EctoHasManyTest.Invoice
```

Comment:
I've added an explicit case, though the simple addition of the `has_many` will make every test fail without the corresponding fix in `put_assoc_changes`.